tests/Feature/Settings/ShowSetUpPageTest.php

```diff
@@ -20,6 +34,10 @@ protected function setUp(): void
 
     protected function getSetUpPageResponse(): TestResponse
     {
+        if ($this->preventStrayRequest) {
+            Http::fake([URL::to('.env') => Http::response(null, 404)]);
```

Comment:
This is only for `testWillCheckDotEnvFileVisibility` and `testWillNotShowErrorWhenDotEnvFileIsNotAccessibleViaHttp` right? Can we inline this into those methods and remove it here so it is obvious that the request is being faked when looking at the test in isolation?

Reply:
Actually, Every testCase for the setup page needs to fake the request to prevent making an actual request which could take  up to 10 seconds when a server is not running. I added it as a little hack to prevent writing Http::fake() in all the the methods